pkg/vm/engine/disttae/database.go

```diff
@@ -57,6 +58,9 @@ func (db *database) Relation(ctx context.Context, name string) (engine.Relation,
 	}
 	tbl, defs, err := db.txn.getTableInfo(ctx, db.databaseId, name)
 	if err != nil {
+		if moerr.IsMoErrCode(err, moerr.OkExpectedEOB) {
```

Comment:
Why is it such a logic. Why does it turn into a parse error here.

Reply:
If a query hit plan cache and it select a table doesn't exist, it will skip the error check in [`pkg/sql/plan/query_builder.go` line 1638](https://github.com/matrixorigin/matrixone/blob/27f506c6d8d7642233412aaf03246435629d72b4/pkg/sql/plan/query_builder.go#L1754)